src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/velocity/mode/VelocityJumpReset.kt

```diff
@@ -77,9 +77,15 @@ internal object VelocityJumpReset : VelocityMode("JumpReset") {
     }
 
     private fun isCooldownOver(): Boolean {
+        val hitsUntilJump = JumpByReceivedHits.hitsUntilJump.random()
+        val ticksUntilJump = JumpByDelay.ticksUntilJump.random()
```

Comment:
This will ranzomize on every tick, which means the chance of getting the minimum value is MUCH higher than the maximum value. Make a variable to store the randomized value in which will be randomized on jump instead.